unblob/handlers/compression/lzh.py

```diff
@@ -0,0 +1,106 @@
+import io
+from typing import List, Optional
+
+from structlog import get_logger
+
+from ...file_utils import Endian
+from ...models import StructHandler, ValidChunk
+
+logger = get_logger()
```

Comment:
This is also not used. If we don't log anything, don't deal with the logging at all.

Reply:
Removed.